www/class/centreonDB.class.php

```diff
@@ -390,4 +392,50 @@ public function getProperties()
         }
         return $info;
     }
+
+    /**
+     * As 'ALTER TABLE IF NOT EXIST' queries are supported only by mariaDB (no more by mysql),
+     * This method check if a column was already added in a previous upgrade script.
+     *
+     * @param string $table - the table on which we'll search the column
+     * @param string $column - the column name to be checked
+     * @return int
+     */
+    public function isColumnExist(string $table = null, string $column = null): int
+    {
+        if (!$table || !$column) {
+            $this->log->insertLog(2, 'UPGRADE PROCESS : Missing data to check if column exist');
```

Comment:
IMHO as you are within the global CentreonDB class, logs should be handled by the calling code, not directly by the class itself.

Reply:
removed ;)